src/Qir/Runtime/unittests/QirRuntimeTests.cpp

```diff
@@ -979,7 +979,7 @@ TEST_CASE("Tuples: copy elision", "[qir_support]")
 // Adjoints for R and Exp are implemented by qis, so let's check they at least do the angle invertion in adjoints.
 struct AdjointsTestSimulator : public SimulatorStub
 {
-    intptr_t lastId      = -1;
+    QubitIdType lastId      = -1;
     double rotationAngle = 0.0;
     double exponentAngle = 0.0;
```

Comment:
Unaligned consecutive assignment. Does the clang-format let it pass during the compilation?

Reply:
Nope, it did indeed fail. Side effect of accepting the updates directly from github ;).